docs/docsite/rst/installation_guide/installation_distros.rst

```diff
@@ -10,23 +10,45 @@ The following instructions will guide you through installing the ``ansible`` pac
 .. contents::
   :local:
 
-Installing Ansible on Fedora or CentOS 
---------------------------------------
+Installing Ansible on Fedora or Enterprise Linux
```

Comment:
Enterprise linux is too loaded a term. We shouldn't include RHEL at all because that has its own installation instructions.

Reply:
Hmm, EPEL is *Extra Packages for **Enterprise Linux***. However, I have an idea on how to rearrange this to avoid that problematic term.